NGitLab/Impl/Utils.cs

```diff
@@ -52,6 +52,9 @@ public static string AddOrderBy(string url, string orderBy = null, bool supportK
 
         private static string AddParameterInternal(string url, string parameterName, string stringValue)
         {
+            if (string.IsNullOrEmpty(stringValue))
```

Comment:
```suggestion
            if (stringValue is null)
```

For methods handling parameters such as `QueryHelpers.AddQueryString`, it's common to only consider null values
- null => don't add the parameter
- empty or any other value => add the parameter

Reply:
Oops, my bad, this is a remnant from a test. Reverted.